2015-Fall/soerendalby/Calculater/ViewController.swift

```diff
@@ -26,5 +27,53 @@ class ViewController: UIViewController {
         
     }
     
+    @IBAction func operate(sender: UIButton) {
+        let operation = sender.currentTitle!;
+        
+        if continueCurrentNumber {
+            enter();
+        }
+        
+        var result:Double?;
```

Comment:
This variable might not be strictly necessary. Why not let the result of ``calculatorBrain.performOperation`` be assigned to displayValue?